### PR TITLE
[scippp] bump version to 1.1.0

### PR DESCRIPTION
**Pull request by hedtke**
Specify library name and version:  **scippp/1.1.0**

I am the author of the library. I just released v1.1.0

One dependency was also bumped in upstream: SCIP to 8.0.4: https://github.com/scipopt/SCIPpp/commit/236dec8b700e281cef5d89744ce587cb5fb3cbfd

Minimum compiler version updated, see upstream https://github.com/scipopt/SCIPpp/pull/15

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by TiloW — APPROVED** [review]

**Comment by hedtke** [comment]
I need help. "This commit cannot be built" but there is no error message

**Comment by hedtke** [comment]
@RubenRBS can I ask for your help here? Where is the problem?

**Comment by RubenRBS** [comment]
Pinging @uilianries, as I'm currently on vacation :)

File: recipes/scippp/all/conanfile.py [hunk 1]
```diff
@@ -34,9 +34,9 @@ def _min_cppstd(self):
     @property
     def _compilers_minimum_version(self):
         return {
-            "gcc": "7",
+            "gcc": "8",
```
Comment (uilianries) [inline recipes/scippp/all/conanfile.py:37]: I'm curious why did you need to change the minimum compiler version? In case this new version is not compatible (e.g. required C++ standard), it should be validated by package version instead. Otherwise, users will lose access to older versions of this library.

Comment (hedtke) [reply recipes/scippp/all/conanfile.py]: Indeed the old versions works fine with the older compiler. The new compiler is now only enforced for the new version.

**Review by uilianries — APPROVED** [review]
LGTM

File: recipes/scippp/all/conanfile.py (hunk 1 above)
Comment (uilianries) [reply recipes/scippp/all/conanfile.py]: Thank you for checking it!

**Review by jwillikers — APPROVED** [review]
